src/Lean/Compiler/LCNF/Testing.lean

```diff
@@ -0,0 +1,154 @@
+/-
+Copyright (c) 2022 Henrik Böving. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Henrik Böving
+-/
+import Lean.Compiler.LCNF.PassManager
+import Lean.Compiler.LCNF.PrettyPrinter
+
+namespace Lean.Compiler.LCNF
+
+partial def Code.containsConst (constName : Name) (code : Code) : Bool :=
+  match code with
+  | .let decl k => goExpr decl.value || containsConst constName k
+  | .fun decl k => containsConst constName decl.value || containsConst constName k
+  | .jp decl k => containsConst constName decl.value || containsConst constName k
+  | .jmp _ args => args.any goExpr
+  | .cases cs => cs.alts.any goAlt
```

Comment:
We can use `cs.alts.any fun alt => containsConst alt.getCode` and delete `goAlt`.